.github/label-actions.yml

```
@@ -124,4 +124,39 @@
     Good luck,
 
 
+    The Renovate team
+
+'pr:no-coverage-ignore':
+  comment: >
+    Hi there,
+
+
+    You're skipping code tests via `istanbul ignore`.
+    Please only do this if it's not possible to test that or it's very difficult to test and won't usually happen at runtime.
```

```suggestion
    Please only skip tests if:
      - you cannot test the code, or
      - the code is very difficult to test, or
      - the edge case won't usually happen at runtime
```

How about putting the exceptions in a list?

Are you sure you want to ignore things that "won't usually happen at runtime"? It sounds to me like this can backfire on us, when we think "oh this code is okay, no need to test", and then you deploy and the server's on fire... 🙈 😄